Makefile

```diff
@@ -40,7 +40,7 @@ install-gofumpt:
 
 .PHONY: install-buf
 install-buf:
-	go install github.com/bufbuild/buf/cmd/buf@v1.31.0
+	go install github.com/bufbuild/buf/cmd/buf@v1.32.0
```

Comment:
This commit will be dropped before merged, it's going to go in separately in https://github.com/polymerdao/monomer/pull/133